src/health/endpoints.rs

```diff
@@ -49,12 +50,13 @@ pub(crate) async fn readyz(
 ) -> Result<String, AppError> {
     let mut acc = MessageAccumulator::new_with_enabled(params.contains_key(VERBOSE_PARAM));
 
-    let mut healthy = true;
-    healthy &= check_sync_state_complete(&state, &mut acc);
-    healthy &= check_epoch_up_to_date(&state, &mut acc);
-    healthy &= check_rpc_server_running(&state, &mut acc).await;
+    let mut ready = true;
```

Comment:
How about 
```rust
let ready = [check_sync_state_complete(&state, &mut acc),
check_epoch_up_to_date(&state, &mut acc),
check_rpc_server_running(&state, &mut acc).await,
check_eth_mapping_up_to_date(&state, &mut acc),
].iter().fold(true, |acc, &v| acc && v);
```

Reply:
What's the benefit of this approach? It seems more readable (severely less readable if you're not proficient in Rust) and, unless Rust does some optimizations under the hood, less performant (though it's negligible in this case).

Reply:
I agree with Hubert. I think we should keep it like this.